docs/spec_draft.md

```diff
@@ -1854,11 +1856,12 @@ not accessed.
 
 ### Constraints
 
-  * (C1) `a` and `b` have the same type and rank $\ge$ 2.
+  * (C1) `a`, `b`, and `result` have the same element type and rank $\ge$ 2.
```

Comment:
(C1) and (C4) has overlapping information. 
(C4) imples that `b` and `result` have the same element type and rank >=2.

I would prefer
(C1) `a` and  `b` have the same element type and rank $\ge$ 2.
(C4)  `b` and `result` have the same type.